### PR TITLE
Chat

### PR DESCRIPTION
**Pull request by JANKROL123**

### CODE REVIEW

File: frontend/src/pages/FriendsPage.tsx [hunk 1]
```diff
@@ -20,6 +23,8 @@ function FriendsPage() {
   const [friendToDelete, setFriendToDelete] = useState<User | null>(null);
 
   const { user } = useUser();
+  const navigate = useNavigate();
```
Comment (Karol-2) [inline frontend/src/pages/FriendsPage.tsx:26]: remove this useNavigate, it's already in line 18

**Review by Karol-2 — APPROVED** [review]
image
Perfecto

**Review by Piterson25 — APPROVED** [review]
Good Job!

**Review by gf-rog — APPROVED** [review]
Nice job, me!

**Pull request merged**